openedx/core/djangoapps/content_libraries/api/container_metadata.py

```diff
@@ -145,3 +379,22 @@ def library_container_locator(
         container_type=container_type.value,
         container_id=container.publishable_entity.key,
     )
+
+
+def get_container_from_key(container_key: LibraryContainerLocator, isDeleted=False) -> Container:
```

Comment:
```suggestion
def get_container_from_key(container_key: LibraryContainerLocator, is_deleted=False) -> Container:
```
Not sure why the linter didn't catch this, but we should use snake_case for parameters

Reply:
Done: 9d808fd43923d0a90b0b184ba69644a683e132ba